mopro-ffi/README.md

```diff
@@ -1,8 +1,27 @@
 # mopro-ffi
 
-This is the main/only crate that is used to build mopro apps.
+Mopro is a toolkit for ZK app development on mobile. Mopro makes client-side proving on mobile simple.
 
-## `SwiftBindings`
+## Getting started
+
+- Make sure you've installed the [prerequisites](https://zkmopro.org/docs/prerequisites).
+- Getting started with this [tutorial](https://zkmopro.org/docs/getting-started/rust-setup).
+
+## Run tests
+
+- circom
+  ```sh
+  cargo test --features circom --no-default-features
```

Comment:
I think we don't need `--no-default-features` in these calls. Could be good to remove that flag and simplify these commands.